addons/web/static/src/js/views/kanban/kanban_renderer.js

```diff
@@ -415,7 +421,11 @@ var KanbanRenderer = BasicRenderer.extend({
 
         // append ghost divs to ensure that all kanban records are left aligned
         var prom = Promise.all(self.defs).then(function () {
-            return self._renderGhostDivs(fragment, 6);
+            var options = {};
+            if (kanbanRecord && kanbanRecord.$el) {
```

Comment:
`kanbanRecord.$el` is always truthy if `kanbanRecord` is truthy.

Reply:
not if the widget isn't started yet

Reply:
but in this case we wait for it so ok

Reply:
:no_mouth: